package.json

```diff
@@ -1151,6 +1153,11 @@
                     "command": "aws.deploySamApplication",
                     "when": "isFileSystemResource == true && resourceFilename =~ /^template\\.(json|yml|yaml)$/",
                     "group": "z_aws@1"
+                },
+                {
+                    "command": "aws.uploadLambda",
+                    "when": "isFileSystemResource == true && resourceFilename =~ /^template\\.(json|yml|yaml)$/ || (explorerResourceIsFolder && isCloud9)",
```

Comment:
parens (grouping conditions) are not supported in vscode package.json syntax, unless that changed recently?

Reply:
Don't see it documented anywhere so no clue
You'd think it'd be supported though. AND should be evaluated first so they aren't needed here.